build/gitversion.yml

```diff
@@ -0,0 +1,35 @@
+assembly-versioning-scheme: MajorMinorPatch
+mode: ContinuousDeployment
```

Comment:
Don't we want to use proper verisionning (similar to chinook packages) 

references: https://github.com/nventive/Chinook.Navigation/blob/main/build/gitversion.yml

Reply:
CC @jeanplevesque

Reply:
We could, we are based on `BiometryService` and other similar packages.

Reply:
Use this for now and we'll switch it once we have a first stable version.